Mathlib/Algebra/Group/Int/Units.lean

```diff
@@ -87,4 +87,16 @@ lemma isUnit_add_isUnit_eq_isUnit_add_isUnit {a b c d : ℤ} (ha : IsUnit a) (hb
 lemma eq_one_or_neg_one_of_mul_eq_neg_one (h : u * v = -1) : u = 1 ∨ u = -1 :=
   Or.elim (eq_one_or_neg_one_of_mul_eq_neg_one' h) (fun H => Or.inl H.1) fun H => Or.inr H.1
 
+instance : Neg ℤˣ where neg u := ⟨-u, -u⁻¹, by simp, by simp⟩
```

Comment:
Do you think we should add, in the appropriate file about the units of a ring, an `example` that shows there there is no non-defeq diamond for `Neg ℤˣ`? This is simple enough that I'm tempted to say it's okay without it, but still I prefer these sorts of sanity checks.

Reply:
Good idea, but the natural place to put such a test would have been the file I'm deleting here, namely `Algebra.Ring.Int.Units`. I'll skip for the time being

bors merge